### PR TITLE
Fixing #71: Add icon of corresponding platform entities in user activity on specific date.

### PR DESCRIPTION
**Pull request by ajuancer**
Issue #71 suggested show the icon of the platform instead of the name. This request also turns the status to an icon. Because maybe it looks a bit confusing, both the status icon and platform icon contains a hover property to show the text (name of the platform or status), with the same style that has the navigation buttons. The platform, besides, redirects to his web when it's pressed. Also a minor change, instead of showing `user activity on <date>`, it's replaced by `<real name> on <date>`.

The code has been tested on Firefox 71.0 and Chrome 79.0.3945.88.

### CODE REVIEW

**Comment by rammanoj** [comment]
@ajuancer adding up to @srish comments. I don't think there is need for displaying different icons for different statuses. Refer to #60. Only the merged contributions of gerrit, merged and open contributions of phabricator will be taken into consideration. Within few days all the other statuses will be removed and hence the present change would be inconsistent. So, I hope you can refer the image at #71 once.

Thanks for the effort :)

**Comment by ajuancer** [comment]
I think I've already made all the requested changes. Hopefully, my contribution is useful, but it's a shame not to be able to use the status icons. Thanks for the mentoring!!

File: frontend/WikiContrib-Frontend/src/App.css [hunk 1]
```diff
@@ -116,17 +116,10 @@ h4.accounts {
   transition: 0.3s;
   font-size: 1.5rem;
   color: #000;
-  outline: none;
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/App.css]: Adjusting the input box's outline was part of another issue, and your PR undid the changes of this PR https://github.com/wikimedia/WikiContrib/pull/100/files. Could you put them back? :)

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 2]
```diff
@@ -72,39 +98,41 @@ class Activity extends React.Component {
...
-                          rel="noopener noreferrer"
-                        >
-                          <h3>{obj.heading}</h3>
-                        </a>
-                      )}
-                      <div>
-                        <span style={{ display: 'inline', float: 'left' }}>
-                          <b>PLATFORM:</b> {obj.platform}
-                        </span>
-                        <span style={{ display: 'inline', float: 'right' }}>
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js]: I kind of like how we were displaying the status with the 'Status' keyboard earlier and the status font was light in weight too.

File: frontend/WikiContrib-Frontend/src/components/activity.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { fetchAsynchronous } from './fetch';
 import { commits_by_date } from './../api';
-import { Card, Placeholder, Header } from 'semantic-ui-react';
-
+import { Card, Placeholder, Header, Popup } from 'semantic-ui-react';
+import gerritPlatformIcon from '../img/gerritPlatformIcon.png';
+import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
```
Comment (srish) [inline frontend/WikiContrib-Frontend/src/components/activity.js]: The Phabricator icon was looking better without the circle, like you had it earlier. Right now it appears wider than the Gerrit icon and it is not looking nice.

**Comment by srish** [comment]
Thanks a lot for your contributions @ajuancer :) There was only a minor issue that I've addressed in this commit here https://github.com/wikimedia/WikiContrib/commit/c753ce73e9d3c9de3bfd636c4865b860c5a17dce.

**Pull request merged**